cmd/loader.go

```diff
@@ -144,7 +144,11 @@ func parseIATDistribution(cfg *config.LoaderConfiguration) (common.IatDistributi
 func parseYAMLSpecification(cfg *config.LoaderConfiguration) string {
 	switch cfg.YAMLSelector {
 	case "container":
-		return "workloads/container/trace_func_go.yaml"
+		if cfg.VSwarm {
```

Comment:
How would loader react if we try to run vSwarm with Firecracker?

Reply:
We have not tested with firecracker since I have not modified anything that relates to firecracker specifically. For now, it is only for Knative and so I think adding a condition for vSwarm in the firecracker block to exit and throw an error would be okay?

Reply:
Yes, that is sensible. Please also update docs which modes are supported, which are not.